force-app/main/default/classes/DMLOperations.cls

```diff
@@ -47,9 +52,14 @@ public with sharing class DMLOperations {
      * @param industry The industry of the account to be created.
      */
     public static void createAccount(String name, String industry) {
+        
         // Create a new Account and set the fields inline
-
+        Account newAccount = new Account(
+            Name = name,
+            Industry = industry
+        );
         // Insert the Account
+        insert newAccount;
```

Comment:
good work. You could do it all at once if you want. 

  public static void createAccount(String name, String industry) {
        Database.insert(new Account(Name = name, Industry = industry), AccessLevel.SYSTEM_MODE);
    }